search/jdbc.sh

```diff
@@ -0,0 +1,39 @@
+#########################################################################
+# File Name: jdbc.sh
+# Author: xiongfusong
+# mail: xiongfusong@gmail.com
+# Created Time: 三  4/20 00:47:19 2016
+#########################################################################
+#!/bin/bash
+zipfile=elasticsearch-jdbc-2.3.1.0-dist.zip
+river=jdbc
+if [ ! -f $zipfile ];then
+    wget http://xbib.org/repository/org/xbib/elasticsearch/importer/elasticsearch-jdbc/2.3.1.0/$zipfile
+fi
+if [ ! -d $river ];then
+    unzip elasticsearch-jdbc-2.3.1.0-dist.zip
+    mv elasticsearch-jdbc-2.3.1.0 $river
+fi
+bin=$river/bin
+lib=$river/lib
+echo '{
+    "type" : "jdbc",
+    "jdbc" : {
+        "url" : "jdbc:mysql://localhost:3306/cavat",
+        "user" : "common",
```

Comment:
I think so, this is only a demo. I am planning to write a script file specially to import elasticsearch from mysql.